project/ISSUE-TRIAGE.md

```diff
@@ -101,9 +106,8 @@ class="gh-label expert">exp/expert</strong> level task.
 
 ### 3. Prioritizing issue
 
-When attached to a specific milestone, an issue can be attributed one of the
-following labels to indicate their degree of priority (from more urgent to less
-urgent).
+When, and only when, attached to a specific milestone, an issue can be attributed one of the
```

Comment:
When, and only when, an issue is attached to a specific milestone, the issue can be labeled with the following...